lib/huk_load/huk_load.c

```diff
@@ -0,0 +1,100 @@
+#include <sys/printk.h>
+#include <pm_config.h>
+#include <string.h>
+
+#include <hal/nrf_acl.h>
+#include <hal/nrf_ficr.h>
+
+#include <nrf_cc3xx_platform.h>
+#include <nrf_cc3xx_platform_mutex.h>
+#include <nrf_cc3xx_platform_abort.h>
+/* The nrf52840 doesn't have a KMU but the helper functions handling the KDR
+*  key is inside the following header */
+#include <nrf_cc3xx_platform_kmu.h>
+
+/* The magic data identify the flash page which contains the key */
+static uint8_t magic_data[16] = { 0x56, 0xB5, 0x6D, 0xAD,
```

Comment:
I would prefer moving the 4 first bytes to the next line with one tab indentation instead, as it would be much more readable

magic_data[16] = {
        xx, xx, xx, xx
        xx, xx, xx, xx
        ...

Reply:
I suggest using a more specific name, e.g. huk_magic.

Reply:
Regarding @frkv s comment you could try to use the clang-format which we have in place in this repo.

Reply:
Will change the name and style, thanks!